dev-docs/conventions.md

```diff
@@ -130,7 +130,7 @@ the `<REPOSITORY>/.vscode/settings.json` repo, so the settings will only affect
     // Attention! This leads to integration test being executed when
     // running a package test within a package containing integration
     // tests.
-    "go.buildTags": "integration",
+    "go.buildTags": "e2e, integration",
```

Comment:
```suggestion
    "go.buildTags": "e2e,integration",
```
I think there cant be a space. At least my VS Code throws a bunch of errors if my string contains white spaces

Reply:
Also might want to update the comment to included a warning that this will cause tests tagged with `e2e` to be run